src/main/java/com/manardenza/entity/Reservation.java

```diff
@@ -3,11 +3,12 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 
+import java.io.Serializable;
 import java.util.Date;
 
 @Data
 @AllArgsConstructor
-public class Reservation {
+public class Reservation implements Serializable {
```

Comment:
If you have marked AbstractObject as Serializable, this class will become so as well